Formula/gor.rb

```diff
@@ -7,6 +7,11 @@ class Gor < Formula
   license "LGPL-3.0"
   head "https://github.com/buger/goreplay.git"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

The latest version matched by the standard regex for Git tags like `1.2.3`/`v1.2.3` is the same as the "latest" release on GitHub (`1.2.0`), so we can use `Git` until it becomes necessary to use `GithubLatest` to be able to find the correct latest version.

Reply:
👍